quinn-proto/src/crypto/rustls.rs

```diff
@@ -40,8 +40,15 @@ impl crypto::Session for TlsSession {
     }
 
     fn early_crypto(&self) -> Option<Self::Keys> {
-        self.get_early_secret()
-            .map(|secret| Crypto::new_0rtt(secret))
+        let secret = self.get_early_secret()?;
+        let suite = self.get_negotiated_ciphersuite().unwrap();
```

Comment:
Do we need this `unwrap()`? In what condition could it occur?

Reply:
Explanatory comment added.